cpp/oneapi/dal/backend/primitives/blas/misc.hpp

```diff
@@ -18,11 +18,24 @@
 
 #include "oneapi/dal/backend/primitives/ndarray.hpp"
 
-#include <mkl_dal_sycl.hpp>
+#include <oneapi/mkl.hpp>
 
 namespace oneapi::dal::backend::primitives {
 
-namespace mkl = oneapi::fpk;
+namespace mkl = oneapi::mkl;
+
+#ifdef ONEDAL_DATA_PARALLEL
+/// Convert oneDAL `ndorder` to oneMKL `layout`
+inline constexpr mkl::layout order_as_layout(ndorder order) {
+    return (order == ndorder::c) ? mkl::layout::R /* row-major */
+                                 : mkl::layout::C /* column-major */;
```

Comment:
```suggestion
    constexpr bool is_rm_order = order == ndorder::c;
    return is_rm_order ? mkl::layout::R /* row-major */
                                 : mkl::layout::C /* column-major */;
```

Usage of explcity naming reduces need for comments